### DISCUSSION TITLE
Extending functionality of manual sorting.

### DISCUSSION THREAD

**Discussion by yarick2000** [directus/directus#6416] [Feature Requests]
I'm thinking about extending default functionality for manual sorting (drag & drop) with additional one that will allow to move groups of multiply selected items to a specific position (before or after target item). That can be done by showing context menu when user making a right click on the target item or when hovering over the sorting column. Here is an example of such context menu: 
![image](https://user-images.githubusercontent.com/13006578/122840806-d55f0b00-d2c8-11eb-8d42-4e78f6923711.png)
User can specify where he wants to place selected ones - before or after targeted item by choosing a related item in the menu. Also, as an additional option, he can specify directly by using input field at which specific position he would like to place an items.

**Comment by benhaynes** [comment]
I like this concept, and it could be useful for manually sorting across pagination. I imagine you would select one or more "source" items, and then navigate to a "destination" item and use right click or the context menu to shift them into an adjacent position. Correct?

**Reply by yarick2000** [reply]
Yes, this is correct

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
